sentinel/src/lib/feed.js

```diff
@@ -22,8 +22,7 @@ const followFeed = (seq, queue) => {
       logger.info(`Sentinel caught up to ${seq}`);
     }
   });
-  feed.follow();
-  return feed;
+  return feed.follow();
```

Comment:
This seems unrelated and possibly a breaking change. Does feed.follow return the feed?

Reply:
Yes. That's how we had it. changesFeed.stop only works with this update.